mobify/platform-scaffold
# AMP-53: add postcss

Added postcss to work with autoprefixer. 

 **JIRA**: https://mobify.atlassian.net/browse/AMP-53
 **Linked PRs**: n/a

## Changes
- postcss
- autoprefixer
- include utilities styles in global file

## How to test-drive this PR
- `npm i`
- `cd node_modules/mobify-amp-sdk`
- `npm i && npm run dev:build`
- `cd ../..`
- Run `npm run dev`
- Preview http://localhost:3000/
- Check make sure it looks fine and include prefixes
